dev/serve.vue

```diff
@@ -17,8 +17,23 @@ export default defineComponent({
 
 <template>
   <div id="app">
-    <cc-explore />
-    <cc-global-header />
-    <cc-global-footer />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@creativecommons/vocabulary@2020.11.3/css/vocabulary.css"
+    />
+    <cc-explore
+      donation_url="https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=top-of-page-banner"
+    />
+    <cc-global-header
+      base_url="http://localhost:8000/wp-json/ccnavigation-header/menu"
```

Comment:
A base URL is typically the part of the address that doesn't change when navigating a site, such as `creativecommons.org`:

`<protocol>://<base_url>/<path>`

So in the example case, the `base_url` would be `localhost:8080`.

That said, we don't need to take such a strict "definition" of "base URL," nor do we need to use `base_url` as the property name. I'm just pointing out a convention that we can consider when designing the component properties.

Related:
- [What is a base URL? ](https://webtech.training.oregonstate.edu/faq/what-base-url)

Reply:
:smile: okay I get it. Just this base URL changes be it in dev, or staging or in production.